module/programs/helper.js

```diff
@@ -509,29 +520,98 @@ module.exports = class ProgramsHelper {
 					}
 				}
 
+				let updateObject = { $addToSet: {} }
+				let validationExcludedEntitiesKeys = []
+				if (
+					userDetails.userInformation.roles.includes(CONSTANTS.common.ADMIN_ROLE) ||
+					userDetails.userInformation.roles.includes(CONSTANTS.common.TENANT_ADMIN)
+				) {
+					// Fetch tenant details to validate organization codes
```

Comment:
@MallanagoudaB we have to throw error if user service call fails. please handle error case